runtime/lua/vim/lsp/buf.lua

```diff
@@ -582,8 +576,11 @@ function M.code_action(context)
   if not context.diagnostics then
     context.diagnostics = vim.lsp.diagnostic.get_line_diagnostics()
   end
-  local params = util.make_range_params()
-  params.context = context
+  local params = function(client)
+    local _params = util.make_range_params(0, client.offset_encoding)
+    _params.context = context
+    return _params
+  end
   code_action_request(params)
```

Comment:
Unless I'm missing something `code_action_request` (and in turn `buf_request_all`) still take regular params, not a function. This looks like it is broken now?

Reply:
Thanks for catching this, should be fixed now.